app/src/test/java/com/codesoom/assignment/application/TaskServiceTest.java

```diff
@@ -0,0 +1,110 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.TaskNotFoundException;
+import com.codesoom.assignment.models.Task;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@DisplayName("TaskService 클래스")
+class TaskServiceTest {
+
+    private TaskService taskService;
+    private static final String TASK_TITLE = "test";
+    private static final String UPDATE_POSTFIX = "New";
+
+    @BeforeEach
+    void setUp() {
+        taskService = new TaskService();
+
+        Task task = new Task();
+        task.setTitle(TASK_TITLE);
+        taskService.createTask(task);
+    }
+
+    @Nested
+    @DisplayName("getTask는 ")
```

Comment:
```suggestion
    @DisplayName("getTask는")
```

여기 빈 칸은 없어도 될 것 같아요. 그리고 아마도 읽기 자연스럽게 작성하기 위해서 `getTasks는` 이라고 붙이신 것 같군요. 
이렇게 사용하면 테스트 결과에서 읽기 좋게 출력되나요?

Reply:
getTasks 메소드라고 하는게 좋을까요?